shenyu-common/src/main/java/org/apache/shenyu/common/config/ShenyuConfig.java

```diff
@@ -1549,7 +1549,12 @@ public static class WebsocketConfig {
          * max frame pay load size mb.
          */
         private Integer maxFramePayloadSize = 10;
-    
+
+        /**
+         * whether enable ping.
+         */
+        private Boolean enablePing = false;
```

Comment:
may `enableProxyPing` adapter the scene?

Reply:
ok, will do thank you.